drgn/commands/_builtin/crash/ps.py

```diff
@@ -0,0 +1,294 @@
+# Copyright (c) 2025 Oracle and/or its affiliates
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""
+Implements the crash "ps" command for drgn.
+
+This command displays process status for selected, or all, processes
+in the system.
+"""
+
+import argparse
+from typing import Any, List, Optional, Sequence, Tuple
+
+from drgn import Object, Program
+from drgn.commands import argument, drgn_argument, mutually_exclusive_group
+from drgn.commands.crash import CrashDrgnCodeBuilder, crash_command
+from drgn.helpers.common.format import (
+    CellFormat,
+    double_quote_ascii_string,
+    escape_ascii_string,
+    print_table,
+)
+from drgn.helpers.linux.kthread import task_is_kthread
+from drgn.helpers.linux.mm import get_task_rss_info, task_vsize, totalram_pages
```

Comment:
FYI I renamed `get_task_rss_info()` to `task_rss_info()`, so you'll want to rebase and update that.